packages/product-components/src/components/AssetLogo/AssetLogo.stories.tsx

```diff
@@ -24,15 +24,26 @@ export const AssetLogo: StoryObj<AssetLogoProps> = {
         contractAddress: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
         shouldTryToFetch: true,
         placeholder: 'USDC',
+        showNetworkIcon: false,
         ...getFramePropsStory(allowedAssetLogoFrameProps).args,
     },
     argTypes: {
+        ...getFramePropsStory(allowedAssetLogoFrameProps).argTypes,
         size: {
             options: allowedAssetLogoSizes,
             control: {
                 type: 'select',
             },
         },
-        ...getFramePropsStory(allowedAssetLogoFrameProps).argTypes,
+        showNetworkIcon: {
```

Comment:
if loading image fails or `shouldTryToFetch` is false, this network icon is not used

<img width="1367" height="430" alt="Screenshot 2025-11-28 at 13 54 36" src="https://github.com/user-attachments/assets/33d347c5-2432-4c6c-90ec-fff90c976049" />

Reply:
✔️